client/state/comments/actions.js

```diff
@@ -125,79 +73,13 @@ export function removeComment( siteId, postId, commentId ) {
  * @param {Number|undefined} parentCommentId parent comment identifier
  * @returns {Function} a thunk that creates a comment for a given post
  */
-export function writeComment( commentText, siteId, postId, parentCommentId ) {
-	if ( ! commentText || ! siteId || ! postId ) {
-		return;
-	}
-
-	return ( dispatch ) => {
-		const placeholderComment = createPlaceholderComment( commentText, postId, parentCommentId ),
-			skipSort = !! parentCommentId;
-
-		// Insert a placeholder
-		dispatch( {
-			type: COMMENTS_RECEIVE,
-			siteId,
-			postId,
-			comments: [ placeholderComment ],
-			skipSort
-		} );
-
-		let apiPromise;
-
-		if ( parentCommentId ) {
-			apiPromise = wpcom.site( siteId ).post( postId ).comment( parentCommentId ).reply( {}, commentText );
-		} else {
-			apiPromise = wpcom.site( siteId ).post( postId ).comment().add( commentText );
-		}
-
-		return apiPromise.then( ( comment ) => {
-			// remove the placeholder
-			dispatch( {
-				type: COMMENTS_REMOVE,
-				siteId,
-				postId,
-				commentId: placeholderComment.ID
-			} );
-
-			// insert the real comment
-			dispatch( {
-				type: COMMENTS_RECEIVE,
-				siteId,
-				postId,
-				comments: [ comment ],
-				skipSort
-			} );
-
-			const requestId = createRequestId( siteId, postId, {} );
-
-			wpcom.site( siteId )
-				.post( postId )
-				.comment()
-				.replies()
-				.then( ( { found: totalCommentsCount } ) => dispatch( {
-					type: COMMENTS_COUNT_RECEIVE,
-					siteId,
-					postId,
-					totalCommentsCount
-				} ) )
```

Comment:
I think this is the part @gwwar was referring to that we are now losing. is it not practical anymore to pull this from the response and dispatch a `COMMENTS_COUNT_RECEIVE` action with the updated data?